addons/website_slides/static/src/js/slides_course_tag_add.js

```diff
@@ -47,6 +47,24 @@ var TagCourseDialog = Dialog.extend({
         });
     },
 
+    /**
+     * Dirty hack to de-activate the "focustrap" from Bootstrap.
+     * Indeed, it prevents typing into our "select2" elements.
+     *
+     * A proper fix would use 'dropdownParent', available in
+     * v4 of select2 (current: v3.5.4), as it allows setting a
+     * dropdown parent instead of appending it at the end of the
+     * body (outside of the modal -> focustrap is an issue).
```

Comment:
As we will drop this fix in saas-17.1 / .2, and we will realistically not upgrade the select2 lib in stable versions, I think we can drop this part of the comment.

That way we keep it straightforward as a temporary and localized fix.